### PR TITLE
Fix engine svg align issue

### PR DESCRIPTION
**Pull request by yiraeChristineKim**
Ref: https://issues.redhat.com/browse/ACM-14573

### CODE REVIEW

**Review by mprahl — APPROVED** [review]

File: frontend/src/routes/Governance/common/util.tsx [hunk 1]
```diff
@@ -732,19 +732,22 @@ export function getEngineWithSvg(apiGroup: string): JSX.Element {
   }
 
   return (
-    <div style={{ display: 'flex', alignItems: 'center', gap: 8 }}>
+    <>
```
Comment (dhaiducek) [inline frontend/src/routes/Governance/common/util.tsx:735]: I didn't know about this--looks like unnamed HTML tags are a thing now? What's the benefit? Is it an alias for `div` or something?

Comment (mprahl) [reply frontend/src/routes/Governance/common/util.tsx:735]: @dhaiducek in React, it's an alias for `React.Fragment` which is a JSX.Element but does not impact the DOM as far as I know. It's just a way to return multiple HTLM elements as a single JSX.Element.

**Pull request merged**